aksel.nav.no/website/.storybook/main.ts

```diff
@@ -0,0 +1,68 @@
+import type { StorybookConfig } from "@storybook/nextjs";
+
+import { readFileSync } from "fs";
+import { join, dirname } from "path";
+import { loadCsf } from "@storybook/csf-tools";
+
+/**
+ * This function is used to resolve the absolute path of a package.
+ * It is needed in projects that use Yarn PnP or are set up within a monorepo.
+ */
+function getAbsolutePath(value: string): any {
+  return dirname(require.resolve(join(value, "package.json")));
+}
+
+const config: StorybookConfig = {
+  storyIndexers: (indexers) => {
+    const csfIndexer = async (fileName: string, opts) => {
+      const code = readFileSync(fileName, "utf-8").toString();
+      return loadCsf(code, { ...opts, fileName }).parse();
+    };
+
+    const exampleIndexer = async (fileName: string, opts) => {
+      let code = readFileSync(fileName, "utf-8").toString();
+
+      code = code
+        .split("\n")
+        .filter((line) => !line.includes("withDsExample"))
+        .join("\n");
+
+      code = code.replace("export const args =", "const args =");
+
+      code += `\nexport default { title: "Eksempler/${fileName
+        .split("pages/eksempler/")[1]
+        .replace(".tsx", "")}"  };\n`;
+
+      return loadCsf(code, { ...opts, fileName }).parse();
+    };
+
+    return [
+      ...(indexers || []),
+      {
+        test: /(stories|story)\.[tj]sx?$/,
+        indexer: csfIndexer,
+      },
+      {
+        test: /pages\/eksempler\/|(".+")|(.+.tsx)$/,
+        indexer: exampleIndexer,
+      },
+    ];
+  },
+  stories: [
+    "../**/*.mdx",
+    "../**/*.stories.@(ts|tsx)",
+    "../pages/eksempler/**/*.tsx",
+  ],
+  addons: [
+    getAbsolutePath("@storybook/addon-essentials"),
```

Comment:
Kan vurdere å skru av actions her også, som i den andre storybooken.

Reply:
-Essientials og -interactions blir brukt i begge nå. Var `@storybook/addon-actions` som ikke trengtes lengre ⚰️

Reply:
Actions er en del av essentials. Men det gjør for så vidt ikke noe at den er der. 
![image](https://github.com/navikt/aksel/assets/83693529/ce7775b2-1ae7-4918-a1e4-09c87dd56b44)